src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx

```diff
@@ -24,7 +23,6 @@ import HelpPopupBody from './HelpPopupBody';
 import { HelpAndDocumentation } from 'src/shared/components/app-bar/AppBar';
 
 import { ReactComponent as HelpIcon } from 'static/img/launchbar/help.svg';
-import { ReactComponent as VideoIcon } from 'static/img/shared/video.svg';
```

Comment:
Should we delete this svg if it is no longer used anywhere?

Reply:
It is used in the popup itself:

![image](https://user-images.githubusercontent.com/6834224/97461561-c8ef0200-1935-11eb-9e87-cf1a3039ee63.png)

Reply:
Ah yes! Sorry.